apps/theme/components/Sidebar/ColorPane/ColorPane.tsx

```diff
@@ -53,19 +58,26 @@ export const ColorPane = ({
         >
           <ChevronLeftIcon title='a11y-title' fontSize='1.5rem' /> Gå tilbake
         </Button>
-        <Button
-          data-size='sm'
-          variant='tertiary'
-          data-color='danger'
-          onClick={() => onRemove()}
-          className={cl(
-            classes.removeBtn,
-            (type !== 'editColor' || colorType === 'neutral') && classes.hide,
-          )}
+        <Tooltip
+          content='Du må ha minst en hovedfarge'
+          hidden={colorType !== 'main' || mainColors.length > 1}
```

Comment:
Couldn't this also use `disableRemoveButton`? It should only show when button is disabled no?

Reply:
right, forgot to change it here. Nice catch